<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::gui::SelectWorkloadDialog</class>
 <widget class="QDialog" name="tt3::gui::SelectWorkloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select workload</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-gui.qrc">
    <normaloff>:/tt3-gui/Resources/Images/Objects/WorkStreamLarge.png</normaloff>:/tt3-gui/Resources/Images/Objects/WorkStreamLarge.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="workloadsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="projectsTab">
      <attribute name="icon">
       <iconset resource="tt3-gui.qrc">
        <normaloff>:/tt3-gui/Resources/Images/Objects/ProjectSmall.png</normaloff>:/tt3-gui/Resources/Images/Objects/ProjectSmall.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Projects</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="projectsFilterWidget" native="true">
         <layout class="QFormLayout" name="formLayout">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="projectsFilterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="projectsFilterLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="projectsTreeWidget">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="showCompletedProjectsCheckBox">
         <property name="text">
          <string>Show completed projects</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="workStreamsTab">
      <attribute name="icon">
       <iconset resource="tt3-gui.qrc">
        <normaloff>:/tt3-gui/Resources/Images/Objects/WorkStreamSmall.png</normaloff>:/tt3-gui/Resources/Images/Objects/WorkStreamSmall.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Work streams</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>2</number>
       </property>
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="workStreamsFilterWidget" native="true">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="workStreamsFilterLabel">
            <property name="text">
             <string>Filter:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="workStreamsFilterLineEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTreeWidget" name="workStreamsTreeWidget">
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="selectionLabel">
     <property name="text">
      <string>&lt;SELECTION&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tt3-gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workloadsTabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_workloadsTabWidgetCurrentChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectsFilterLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_projectsFilterLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workStreamsFilterLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_workStreamsFilterLineEditTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectsTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_projectsTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workStreamsTreeWidget</sender>
   <signal>itemChanged(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_workStreamsTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showCompletedProjectsCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_showCompletedProjectsCheckBoxStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>projectsTreeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_projectsTreeWidgetItemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workStreamsTreeWidget</sender>
   <signal>itemDoubleClicked(QTreeWidgetItem*,int)</signal>
   <receiver>tt3::gui::SelectWorkloadDialog</receiver>
   <slot>_workStreamsTreeWidgetItemDoubleClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_workloadsTabWidgetCurrentChanged(int)</slot>
  <slot>_projectsFilterLineEditTextChanged(QString)</slot>
  <slot>_workStreamsFilterLineEditTextChanged(QString)</slot>
  <slot>_projectsTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_workStreamsTreeWidgetItemChanged(QTreeWidgetItem*,int)</slot>
  <slot>_showCompletedProjectsCheckBoxStateChanged(int)</slot>
  <slot>_projectsTreeWidgetItemDoubleClicked(QTreeWidgetItem*,int)</slot>
  <slot>_workStreamsTreeWidgetItemDoubleClicked(QTreeWidgetItem*,int)</slot>
 </slots>
</ui>
